packages/TextEditor/src/components/Tiny/Tiny.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+// eslint-disable-next-line no-unused-vars
+export default function Tiny(props) {
+  // shell component
+  return <></>;
+}
+
+Tiny.displayName = "TextEditor.Tiny";
+
+Tiny.propTypes = {
+  /** this object can be any of the configuration options support by TinyMCE https://www.tiny.cloud/docs/configure/  */
+  init: PropTypes.shape({}).isRequired,
```

Comment:
Why this wrapper is necessary? Why not just move it to `TextEditor ` ?

Reply:
to keep the way other components behave when modifying third-party libraries in our library, this way helps to bring clarity between paprika component and the tinyMCE props.